torch/_inductor/graph.py

```diff
@@ -148,6 +149,61 @@ def getattr_recursive(obj, target):
     return attr_itr
 
 
+def mark_nodes_dislike_padding(g):
+    """
+    Nodes like convolution/convolution_backward want its input to be contiguous.
+    If we pad their inputs, we result in extra calls to copy kernels!  On the other hand, padding usually helps reduction.
+
+    The pass finds nodes that dislike padding. These are nodes that can be reached
+    from a convolution/convolution_backward in the backward direction without
+    going thru a reduction.
+    """
+    if not config.comprehensive_padding:
+        return
+    ops_dislike_padding = {
+        aten.convolution,
+        aten.convolution_backward,
+    }
+    # what's a better way to collect the reduction ops?
+    ops_like_padding = {
```

Comment:
It would be great (not needed for this PR per se) to have a `reduction` tag similar to how we have a `pointwise` tag. This was also recently a need in @Chillee's aot autograd pr.